examples/access-timeout-meta/README_pt.adoc

```diff
@@ -0,0 +1,287 @@
+= @AccessTimeout la forma  Meta-Annotation 
+:index-group: Meta-Annotations
+:jbake-type: page
+:jbake-status: status=published
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:stylesheet: ../github-stylesheet.css
+endif::[]
+
+== Indrodução
+Qualquer anotação que recebe parâmetros podem se beneficiar das meta-anotações. Aquí observamos como `@AccessTimeout` pode ser mais facíl de entender e administrar mediante meta-anotações.
+Usaremos [access-timeout](../access-timeout/README.html) como nosso exemplo de caso de uso.
+
+O valor dos parâmetros do fornecido para o `@AccessTimeout` tem um efeito dramático em que a anotação realmente faz. Além disso, `@ AccessTimeout` tem um desses designs
+onde `-1` e `0` tem um significado totalmente diferente. Um quer dizer "espere para sempre", o outro "nunca espere". Só alguns com sorte podem lembrar qual é qual.
+Para o resto de nós é uma fonte constante de erros.
+
+TIP: Meta-Anotações. ao resgate!
+
+
+== *_Criando Meta-Anotações._*
+
+Com respeito as melhores práticas, colocaremos nossas meta-anotações em um pacote chamado `api`, para este exemplo que nós da `org.superbiz.accesstimeout.api`. O pacote `org.superbiz.api` também vai funcionar.
+
+
+A ideia básica é ter um pacote onde anotações "aprovadas" sejam usadas e proibir uso de versões não-meta-anotações. Toda a configuração
+estará centralizada no pacote `api` e alterações nos valores de tempo limite estarão localizadas nesse pacote e refletidas automaticamente em todo o aplicativo.
+
+Um efeito secundario interesante desta abordagem é que se o pacote `api` onde as definições de meta-anotação existem localizados em um jar separado, então efetivamente alguem
+pode trocar toda a configuração de uma aplicação simplesmente substituindo o jar `api`.
+
+
+=== @Metatype [.small]#A Meta-Anotação "root"#
+
+Como todo o uso de meta-anotação, primeiro devemos criar mossa própria meta-anotaçaõ "root". Isto é tão facil como criar uma anotação
```

Comment:
in this sentence "primeiro devemos criar mossa própria meta-anotaçaõ", the correct is "nossa" instead of "mossa" and the accent in "anotaçaõ" on the letter "a".